templates/default/event/eventnote.html

```diff
@@ -9,7 +9,7 @@ <H1>{$layout.pagetitle}</H1>
     {block name="event_note_table"}
 	<TR class="dark">
 		<TD width="1%" class="fleftu">
-			<IMG src="img/calendar.gif" alt="">
+			{icon name="calendar"}
 		</TD>
```

Comment:
dlaczego tu bez fw=true?

Reply:
Nie widziałem mankament'ów wizualnie - może zróbmy domyślnie fw=true?

Reply:
@chilek nie widziałem mankament'ów wizualnie - może zróbmy domyślnie fw=true? (jeśli nie to dorzucę wszędzie w tym commicie fw=true). Jak chcesz?

Reply:
Nie prościej byłoby teraz dać wszędzie fw=true, a potem sed-em wszystko wyczyścić mają dzieki temu gwarancję, ze nawet przez chwilę nie będzie takiej sytuacji, że coś dziwnie działa?

Reply:
Ok - zrobię tak.

Reply:
`fw=true` ląduje wszędzie tam, gdzie mamy fa w kolumnie. Inaczej się rozjedzie. Niektóry ikony z fa są szersze.

Reply:
Zrobione. Dodałem też fw=true w brakujących miejscach.